components/esp32/panic.c

```diff
@@ -212,7 +212,10 @@ void panicHandler(XtExcFrame *frame)
                     //debugcause if the cause is watchdog 1 and clearing it if it's watchdog 0.
                     if (debugRsn&(1<<8)) {
 #if CONFIG_FREERTOS_WATCHPOINT_END_OF_STACK
-                        panicPutStr("Stack canary watchpoint triggered ");
+                        const char *name = pcTaskGetTaskName(xTaskGetCurrentTaskHandleForCPU(core_id));
```

Comment:
but task name is a member of the current TCB, so, is it actually possible for it to be not in RAM?